### DISCUSSION TITLE
Bitmart exchange Fundraiser

### DISCUSSION THREAD

**Discussion by sugoibtc** [coinselor/twitter#14] [Tweet Ideas]
💥 BitMart Exchange Fundraiser

Join forces in the community crowdfunding initiative for a native $ZNN listing on @bitmartexchange.

Enhance NoM's global presence with us. For more details on how to contribute, please visit the following link 👇
https://forum.zenon.org/t/bitmart-exchange-zenon-listing-fund-raiser/1774

**Comment by coinselor** [comment]
💥 BitMart Exchange Fundraiser

Join the community's crowdfunding initiative for a native $ZNN listing on @bitmartexchange.

Enhance NoM's global presence with us. For more details on how to contribute, please visit the following link 👇
https://forum.zenon.org/t/bitmart-exchange-zenon-listing-fund-raiser/1774

---

^ few grammar corrections by ChatGPT

How about changing it to `... initiative for native support of ZTS assets on @bitmartexchange` ?

**Reply by sugoibtc** [reply]
That's a nice suggestion! It should fit within the character limit. This is the revisted tweet:

💥 BitMart Exchange Fundraiser

Join forces in the community crowdfunding initiative for native support of #ZTS assets on @BitMartExchange.

Enhance NoM's global presence with us. For more details on how to contribute, please visit the following link 👇
https://forum.zenon.org/t/bitmart-exchange-zenon-listing-fund-raiser/1774

**Reply by coinselor** [reply]
`
Join forces in the community's crowdfunding initiative
`

By changing it to "Join forces in the community's crowdfunding initiative," the addition of the apostrophe and the letter 's' ('s) clarifies that the crowdfunding initiative belongs to or is associated with the community.
